### PR TITLE
fix: crypto is not accessible in a Worker scope

### LINKED ISSUE #5111: crypto is not accessible in a Worker scope
Use Case: I am trying to spawn Workers that connect to the Discord API for each shard. To do this, each worker needs to connect to the websocket. It seems that `crypto` is not accessible inside a Worker so it makes it impossible to use Websockets with Workers.

Error:
image

### PR DESCRIPTION
**Pull request by sunsetkookaburra**
Closes #5111 
move `crypto` to globals/sharedGlobals

### CODE REVIEW

**Comment by sunsetkookaburra** [comment]
oops, 
included wrong commits

File: cli/tests/039_worker_deno_ns.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
-const w1 = new Worker("./039_worker_deno_ns/has_ns.ts", { type: "module" });
+const w1 = new Worker("./039_worker_deno_ns/has_ns.ts", {
+  type: "module",
+  deno: true,
+});
```
Comment (bartlomieju) [inline cli/tests/039_worker_deno_ns.ts:4]: Revert; these files are being removed in #5128; please add a test case to `cli/tests/worker_test.ts`

**Comment by sunsetkookaburra** [comment]
@bartlomieju Do I need to squash the commits into one, or will the PR put them into a single commit?

**Comment by bartlomieju** [comment]
> @bartlomieju Do I need to squash the commits into one, or will the PR put them into a single commit?

That's not needed, we'll squash them when merging

**Comment by sunsetkookaburra** [comment]
this should be ready now

**Pull request merged**